### PR TITLE
Added subset forcings, added missing directories and set flags in yam…

### PR DESCRIPTION
**Pull request by JurgenZach-NOAA**
…l file to make sure the example runs out of the box

[Short description explaining the high-level reason for the pull request]

## Additions

- Subset forcings for date 0401
- Empty directories output and restart

## Removals

-

## Changes

- Changes flags in yaml file: 
             treamflow_nudging                  : True
            diffusive_streamflow_nudging  : True

## Notes

-

## Todos

-

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows project standards (link if applicable)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Target Environment support

- [ ] Windows
- [ ] Linux
- [ ] Browser

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

File: test/LowerColorado_TX_v4/test_AnA_V4_HYFeature.yaml [hunk 1]
```diff
@@ -67,8 +67,8 @@ compute_parameters:
         usace_timeslices_folder  : #usace_timelices/
         streamflow_da:
             #----------
-            streamflow_nudging            : False
-            diffusive_streamflow_nudging  : False                    
+            streamflow_nudging            : True
+            diffusive_streamflow_nudging  : True
```
Comment (shorvath-noaa) [inline test/LowerColorado_TX_v4/test_AnA_V4_HYFeature.yaml:71]: This should be false. We aren't doing diffusive routing for now.

**Comment by shorvath-noaa** [comment]
@JurgenZach-NOAA I don't see the two new subdirectories (restart and output I believe?) Can you add those?

**Comment by JurgenZach-NOAA** [comment]
Corrected both issues: diffusive routing flag and "empty" directories

**Review by shorvath-noaa — APPROVED** [review]

**Comment by hellkite500** [comment]
I would be cautious adding too much test data directly to the repository. This can be cleaned up from the history in the future, but it requires a force push and history re write (this has already happened once in this repos history...)

Large domain and test data of this sort should be hosted outside the code base typically and referenced/pulled into test scripts ect. I'm not sure how large the binary files actually are, but having them in the history starts to impact clone performance significantly.

**Pull request merged**